rorm/rorm-sample/src/operations/prepared_statements.rs

```diff
@@ -0,0 +1,61 @@
+use rorm::{value::Value, Database};
+
+async fn check_select_star(db: &Database, raw_query: &str) -> anyhow::Result<()> {
+    let rows = db.raw_sql(raw_query, None, None).await?;
+    assert_eq!(rows.len(), 5, "created six users but dropped Eve");
+    let vector_of_user_ids: Vec<i64> = rows
+        .iter()
+        .map(|r| r.get::<i64, &str>("id").unwrap())
```

Comment:
Use `r.get("id")` instead of turbofish syntax. The compiler knows the first parameter through the annotation of your resulting variable. The second is inferred through the type you provide as first argument to the method.